Xamarin.Forms.Xaml/XmlnsHelper.cs

```diff
@@ -20,6 +20,19 @@ public static void ParseXmlns(string xmlns, out string typeName, out string ns,
 		{
 			typeName = ns = asm = targetPlatform = null;
 
+			xmlns = xmlns.Trim();
+
+			if (xmlns.StartsWith("using:", StringComparison.Ordinal)) {
+				ParseUsing(xmlns, out typeName, out ns, out asm, out targetPlatform);
+				return;
```

Comment:
avoid unnecessary nesting ? no real reason for one option or the other. If you think I did that to optimize the compiled code, well, you're thinking too much :)